.github/workflows/testPrerelease.yml

```diff
@@ -0,0 +1,21 @@
+name: On Tag
```

Comment:
These are just examples of how this *could* be used. It obviously doesn't need to work this way in the actual repos.